app/views/user_mailer/reset_password_instructions.html.erb

```diff
@@ -1,3 +1,27 @@
+<% if @pending_profile_requires_verification %>
+  <table>
+    <tbody>
+    <tr>
+      <td width="140">
+        <%= image_tag('letter-warning.svg', width: 140, height: 140, alt: '') %>
+      </td>
+      <td class="padding-left-3">
+        <h2 style="font-size:22px;"><%= t('user_mailer.reset_password_instructions.gpo_letter_header') %></h2>
+        <p><%= t('user_mailer.reset_password_instructions.gpo_letter_description') %></p>
+      </td>
+    </tr>
+    <tr>
+      <td class="s30" style="height: 30px"></td>
+    </tr>
+    <tr>
+      <td class="border-top border-primary-light s30" style="margin-top: 3em; margin-bottom: 3em; height: 30px;" colspan="2" align="center"></td>
+    </tr>
+    </tbody>
+  </table>
```

Comment:
I wonder if it might be easier to use a semantic `<hr />` with design system classes, rather than emulating it with a table cell.

```suggestion
  </table>
  <hr class="border-0 border-bottom border-primary-light margin-y-4" />
```